nni/algorithms/compression/pytorch/pruning/iterative_pruner.py

```diff
@@ -3,8 +3,9 @@
 
 import logging
 import copy
+from nni.utils import OptimizeMode
```

Comment:
What is `OptimizeMode` for? seems not used?

Reply:
yes, seems a mistake, remove it.